openvoxserver/Containerfile

```
@@ -76,28 +78,27 @@ COPY docker-entrypoint.d /docker-entrypoint.d
 COPY --from=builder /var/lib/gems/ /var/lib/gems/
 COPY --from=builder /usr/local/bin/r10k /usr/local/bin/
 
-ADD https://s3.osuosl.org/puppet-artifacts/openvox-server/${OPENVOXSERVER_VERSION}/openvox-server_${OPENVOXSERVER_VERSION}-1%2Bubuntu24.04_all.deb /
-ADD https://s3.osuosl.org/puppet-artifacts/openvox-agent/${OPENVOXAGENT_VERSION}/openvox-agent_${OPENVOXAGENT_VERSION}-1%2Bubuntu24.04_${TARGETARCH}.deb /
-ADD https://s3.osuosl.org/puppet-artifacts/openvoxdb/${OPENVOXDB_VERSION}/openvoxdb-termini_${OPENVOXDB_VERSION}-1%2Bubuntu24.04_all.deb /
+ADD https://apt.overlookinfratech.com/openvox${OPENVOX_RELEASE}-release-ubuntu${UBUNTU_VERSION}.deb /
+RUN apt-get update && \
+    apt-get install -y ca-certificates && \
+    dpkg -i /openvox${OPENVOX_RELEASE}-release-ubuntu${UBUNTU_VERSION}.deb && \
+    rm /openvox${OPENVOX_RELEASE}-release-ubuntu${UBUNTU_VERSION}.deb
 
 RUN groupadd -g ${OPENVOX_USER_GID} puppet && \
     useradd -m -u ${OPENVOX_USER_UID} -g puppet puppet && \
     chmod +x /docker-entrypoint.sh /healthcheck.sh /docker-entrypoint.d/*.sh && \
     apt-get update && \
     apt-get upgrade -y && \
     apt-get install -y $PACKAGES && \
-    dpkg -i /openvox-agent_${OPENVOXAGENT_VERSION}-1+ubuntu24.04_${TARGETARCH}.deb && \
-    dpkg -i /openvox-server_${OPENVOXSERVER_VERSION}-1+ubuntu24.04_all.deb && \
-    dpkg -i /openvoxdb-termini_${OPENVOXDB_VERSION}-1+ubuntu24.04_all.deb && \
+    apt-get install -y openvox-agent=${OPENVOXAGENT_VERSION}-1+ubuntu${UBUNTU_VERSION} && \
+    apt-get install -y openvox-server=${OPENVOXSERVER_VERSION}-1+ubuntu${UBUNTU_VERSION} && \
+    apt-get install -y openvoxdb-termini=${OPENVOXDB_VERSION}-1+ubuntu${UBUNTU_VERSION} && \
```

Do you need to specify the exact package name like this? By hardcoding `-1` you're making sure you can't use releases to patch small things. Also, probably better to run it in a single `apt-get install` command. The server can pull in a different agent version and might up-/downgrade it this way. I suspect that with a single command it'll refuse to run instead. You can probably even combine it with `$PACKAGES`

---

i don't know how to better pin the version. i tried package=version but this didnt work until i added the rest :(.

---

Some experimentation tells me `apt install openvox-agent=8.11.0-*` works for me so I think this should work:
```suggestion
    apt-get install -y openvox-agent=${OPENVOXAGENT_VERSION}-* openvox-server=${OPENVOXSERVER_VERSION}-* openvoxdb-termini=${OPENVOXDB_VERSION}-* && \
```